pkg/controller/metrics/client_metrics.go

```diff
@@ -64,6 +64,10 @@ func (l *latencyAdapter) ClearState() {
 	l.m.Reset()
 }
 
+func (l *latencyAdapter) FQName() string {
+	return l.m.FQName()
```

Comment:
Not sure if these are the proper way to implement the `FQName()` function

Reply:
It seems like the update that brought this requirement (https://github.com/kubernetes/kubernetes/commit/4d028a7903ac66db5795056f654c8b0c03b7c90a) was minor and I think in this case using the metric's provided FQName() function is sufficient